### PR TITLE
Minor changes to prepare for IES2

### PR DESCRIPTION
**Pull request by joakim-hove**
**Issue**
 Attempt to extract changes to existing code from #422 - so that the #422 will - as much as possible - only be completely new code.

### CODE REVIEW

**Comment by joakim-hove** [comment]
I have one more minor commit coming.

**Comment by joakim-hove** [comment]
I am trying to reduce the number of moving parts with respect to #422 - would be good to have this merged.

File: lib/enkf/enkf_main.cpp [hunk 1]
```diff
@@ -70,6 +70,7 @@
 #include <ert/res_util/res_util_defaults.hpp>
 #include <ert/res_util/subst_func.hpp>
 #include <ert/res_util/res_log.hpp>
+#include <ert/res_util/es_testdata.hpp>
```
Comment (markusdregi) [inline lib/enkf/enkf_main.cpp]: Are there any programmatic changes to this file? If not, I suggest to remove the newly added scope to keep the changes small... Please correct me if I'm wrong!

The improved comments are of course very much welcome 👍

File: lib/analysis/modules/rml_enkf.c [hunk 2]
```diff
@@ -620,6 +620,7 @@ void rml_enkf_updateA(void * module_data,
 
 void rml_enkf_init_update(void * arg,
                           const bool_vector_type * ens_mask,
+                          const bool_vector_type * obs_mask,
```
Comment (markusdregi) [inline lib/analysis/modules/rml_enkf.c:623]: I guess I can read this change as this code actually having zero tests?

**Review by markusdregi — COMMENTED** [review]
This looks good to me!

Please consider to remove the block as pointed out above if that is the single cause of the "big" diff.

File: lib/analysis/modules/rml_enkf.c (hunk 2 above)
Comment (joakim-hove) [reply lib/analysis/modules/rml_enkf.c:623]: Not even compiled by default ... :-(

File: lib/enkf/enkf_main.cpp (hunk 1 above)
Comment (joakim-hove) [reply lib/enkf/enkf_main.cpp]: > I suggest to remove the newly added scope to keep the changes small.

Everyone complain about my scopes; when I become dictator then there will be scopes - yessir; but for now I have removed it :-)

**Comment by joakim-hove** [comment]
Give this a Jenkins kick?

File: lib/analysis/modules/rml_enkf.c (hunk 2 above)
Comment (markusdregi) [reply lib/analysis/modules/rml_enkf.c:623]: Scary stuff; robots from the future will come for us soon!

File: lib/enkf/enkf_main.cpp (hunk 1 above)
Comment (markusdregi) [reply lib/enkf/enkf_main.cpp]: Scopes went out of fashion in C99... ;)

**Comment by markusdregi** [comment]
Test this please

**Review by markusdregi — APPROVED** [review]

**Pull request merged**